weave

```diff
@@ -1625,13 +1628,16 @@ launch_router() {
            --label=weavevolumes $WEAVEDB_IMAGE >/dev/null
     fi
 
+    RESOLV_CONF_DIR=$(dirname $(chroot $HOST_ROOT readlink -f /etc/resolv.conf))
```

Comment:
Really I should take the `basename` of this too and pass it into the router - right now we assume that the ultimate file is still called `resolv.conf`. Thoughts?